pyvista/plotting/plotter.py

```diff
@@ -6404,7 +6405,7 @@ def __init__(
                 notebook = scooby.in_ipykernel()
 
         self.notebook = notebook
-        if self.notebook:
+        if self.notebook or pyvista.SCREENSHOTS_ONLY:
```

Comment:
This is a great feature. Thank you. I will follow the `OFF_SCREEN` naming convention and suggest the variable name `ON_SCREENSHOT`.
```suggestion
        if self.notebook or pyvista.ON_SCREENSHOT:
```